src/Shell/Task/ExtractTask.php

```diff
@@ -447,6 +461,9 @@ protected function _parse($functionName, $map)
                         'file' => $this->_file,
                         'line' => $line,
                     ];
+                    if ($this->_relativePaths) {
+                        $details['file'] = '.' . str_replace(ROOT, '', $details['file']);
```

Comment:
If anyone has a better idea for the string replacing here - please just change it! 

Also using the `.` in front of the path is more of a personal experience. I couldn't see Cakephp using either `./` or `/` so I just picked my favorite.

Just tell me if I should change it.

Reply:
I personally would just use `foo/bar/baz.ctp` without any leading dot or slash.

Reply:
I usually replace `ROOT . DS` completely :)
But either way is already relative and works fine.